File: BBIS.Api/ClientApp/src/models/DonorRegistration/MedicalQuestionnaireDto.ts
export interface IMedicalQuestionnaireDto {
    MedicalQuestionnaireId: number,
    HeaderText: string,
    QuestionTagalogText: string,
    QuestionEnglishText: string,
    OrderNo: number,
    GenderOption: string,
}

export class MedicalQuestionnaireDto implements IMedicalQuestionnaireDto{
    MedicalQuestionnaireId: number = 0;
    HeaderText: string = "";
    QuestionTagalogText: string = "";
    QuestionEnglishText: string = "";
    OrderNo: number = 0;
    GenderOption: string = "";
}
  

File: BBIS.Application/DTOs/DonorRegistration/MedicalQuestionnaireDto.cs
namespace BBIS.Application.DTOs.DonorRegistration
{
    public class MedicalQuestionnaireDto
    {
        public int MedicalQuestionnaireId { get; set; }
        public string HeaderText { get; set; }
        public string QuestionTagalogText { get; set; }
        public string QuestionEnglishText { get; set; }
        public string QuestionBisayaText { get; set; }
        public int OrderNo { get; set; }
        public string GenderOption { get; set; }
    }
}
